workflows/edit_eml/edit_semantic_annotation.Rmd

```diff
@@ -1,21 +1,37 @@
 ## Semantic annotations
 
-The training will cover an overview of what areas we usually will need to update. For more information on how to add semantic annotations to EML 2.2.0 can be found
+For a brief overview of what a semantic annotation is, and why we use them check out [this video.](https://drive.google.com/file/d/1Err-fL8O21kd1NzHJ9HJkK_B2fpt4sQW/view?usp=sharing)
+
+Even more information on how to add semantic annotations to EML 2.2.0 can be found
  <a href = 'https://eml.ecoinformatics.org/semantic-annotation-primer.html'>here</a>. Currently metacatUI does not support the editing of semantic annotations on the website so all changes will have to be done in R.
+ 
+There are several elements in the EML 2.2.0 schema that can be annotated:
+
+* `dataset`
+* entity (eg: `otherEntity` or `dataTable`)
+* `attribute`
```

Comment:
There are more than this but I think these three are the key ones for datateam members so this. Which you probably thought of so this is 👍 .